index.html

```diff
@@ -10,10 +11,94 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    
+     
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
-</body>
-</html>
+    <header>
+        <nav>
+          <div class="nav-logo">
+            <img class="karma-logo" src="./img/karma-logo.svg" />
+          </div>
+          <div class="nav-logo nav-logo-right">
+            <a class="nav-link active" href="#">Meet Karma</a>
+            <a class="nav-link" href="#">How It Works</a>
+            <a class="nav-link" href="#">Store</a>
+            <a class="nav-link" href="#">Blog</a>
+            <a class="nav-link" href="#">Help</a>
+            <a class="nav-link" href="#">Login</a>
+          </div>
+        </nav>
+      </header>
+      <main>
+        <section class="Karma">
+          <h1 class="Karma-title">Introducing Karma</h1>
+          <h2 class="Karma-description">
+            Bring WiFi with you, everywhere you go.
+          </h2>
+          <a class="btn-Karma" href="#">Learn More</a>
+        </section>
+
+
+        <section class="info">
+              <h2 class="info-title">Everyone needs a little Karma.</h2>
+              <div class="cards-container">
+                  <div class="card">
+                      <img src="img/icon-devices.svg" />
```

Comment:
These links in `index.html` are now broken, because the `img/` folder has moved from root level, to `/store/img`. This means that when `index.html` looks for `img/pic.svg` it is looking for the `img/` folder in the same level as the `index.html` file, but it's now one level nested to `/store/img`. How would you fix this?

Reply:
fixed now